docs/source/advanced_features/cloud_storage.rst

```diff
@@ -14,7 +14,7 @@
 | In default cloud storage mode, data are stored on TensorBay cloud.
```

Comment:
Rename the examples/ to code/

Add an __init__.py file under docs/.
See examples/__init__.py as an example.